src/logic/safe/store/reducer/__tests__/gatewayTransactions.test.ts

```diff
@@ -399,386 +247,11 @@ describe('gatewayTransactionsReducer', () => {
         4: {
           [ZERO_ADDRESS]: {
             queued: {
-              next: {},
-              queued: {
+              next: {
```

Comment:
This should not pass:
1. The `QUEUED_LABEL` was passed in the payload and these are therefore `queued.queued` transactions.
2. `queued.next` only contains one array of transactions of the same nonce.

Reply:
The code currently doesn't care about what label that is, only about its position.
You think we should check the actual content of the label?
Or should I update the test instead?

Reply:
I would suggest we check the label and add tests accordingly.

Reply:
I'm now searching for the Queued label. Adjusted the test.